custom/enikshay/user_setup.py

```diff
@@ -266,12 +266,6 @@ def connect_signals():
     commcare_user_post_save.connect(save_user_callback, dispatch_uid="save_user_callback")
 
 
-class ENikshayNewMobileWorkerForm(NewMobileWorkerForm):
-    """Mobile worker list view create modal"""
-    def __init__(self, *args, **kwargs):
-        super(ENikshayNewMobileWorkerForm, self).__init__(*args, **kwargs)
-
-
```

Comment:
Why was  `ENikshayNewMobileWorkerForm ` even needed?
You removed it because it was not doing anything special?

Reply:
I think it used to do something which later got moved elsewhere.  At this point it's not doing anything, though.